src/main/java/chess/piece/type/Rook.java

```diff
@@ -0,0 +1,14 @@
+package chess.piece.type;
+
+import chess.piece.type.movable.RookPieceMovable;
+import chess.score.Score;
+import chess.team.Team;
+
+public class Rook extends Piece {
+    private static final char NAME = 'r';
+    private static final int SCORE = 5;
```

Comment:
```suggestion
    private static final Score SCORE = new Score(5);
```

score 타입으로 가지고 있어도 될 것 같습니다 :)

Reply:
하나 여쭤봐도 될까요 ?! 

위의 코드에서 NAME도 Name 이라는 enum으로 래핑할 수 있을 것 같아요! 

기본 타입의 래핑은 **항상** 하는 것이 좋을까요 ?

지금 제 코드 상에서는 Name이 정해진 뒤 다른 클래스에 전달되거나 그런 로직 없이 클래스 내부에서만 사용하고 있습니다.

과도할 수도 있다는 생각이 들어서요!

이럴 경우에도 항상 래핑하는 것이 좋을까요 ?!

Reply:
말씀하신 것 처럼 Name도 enum으로 분리가 가능합니다!
값 객체를 포장하고 일급 컬렉션을 만드는 등 작은 단위로 나누는 것은 정말 좋은데요.
시간이 한정적이기 때문에 일정을 지키는 선에서 나눈다면 매우 좋은 공부가 될 것 같습니다 :)